kornia/augmentation/random_generator/_2d/gaussian_illumination.py

```diff
@@ -0,0 +1,121 @@
+from __future__ import annotations
+
+import torch
+
+from kornia.augmentation.random_generator.base import RandomGeneratorBase, UniformDistribution
+from kornia.augmentation.utils import _adapted_rsampling, _common_param_check, _range_bound
+from kornia.core import Tensor
+from kornia.utils import _extract_device_dtype
+
+
+def gaussian(window_size, sigma, center=None):
```

Comment:
there's an existing function https://github.com/kornia/kornia/blob/main/kornia/filters/kernels.py#L62

Reply:
Yes, I copied it from there but adding a parameter of center.
If you think it's ok to add center we can merge this two functions if it does not change normal behaviour.

Reply:
+1 to add support to center on the gaussian kernel function. If it keeps back-compatibility, and add this feature, I do not see why not 😄